src/components/dialog/demoBasicUsage/script.js

```diff
@@ -57,30 +56,19 @@ angular.module('dialogDemo1', ['ngMaterial'])
   };
 
   $scope.showAdvanced = function(ev) {
-    var useFullScreen = ($mdMedia('sm') || $mdMedia('xs'))  && $scope.customFullscreen;
-
     $mdDialog.show({
       controller: DialogController,
       templateUrl: 'dialog1.tmpl.html',
       parent: angular.element(document.body),
       targetEvent: ev,
-      clickOutsideToClose:true,
-      fullscreen: useFullScreen
+      clickOutsideToClose: true,
+      fullscreen: true // Fullscreen only applies for -xs, -sm breakpoints.
```

Comment:
Sure! There are several other dialogs in this example that don't use fullscreen. My personal opinion is that they were sufficient to show what mdDialog is about without fullscreen. But I'll make that change.